Source/missiles.cpp

```diff
@@ -1664,7 +1660,6 @@ void AddMana(Missile &missile, Point /*dst*/, Direction /*midir*/)
 		player._pManaBase = player._pMaxManaBase;
 	UseMana(missile._misource, SPL_MANA);
 	missile._miDelFlag = true;
-	drawmanaflag = true;

```

Comment:
I assume this is being handled by `UseMana` already and is why we can remove it from here?

Reply:
The spell it self doesn't affect your mana.

Reply:
I'm not sure. Doesn't it change mana?
I tested with `Staff of Mana` and it will not redraw the mana flask correctly.
But perhaps I miss something.

Reply:
Your right, i was thinking this was AddManashield